backend/src/main/java/ai/verta/modeldb/utils/RdbmsUtils.java

```diff
@@ -93,6 +95,12 @@
   private RdbmsUtils() {}
 
   private static final Logger LOGGER = LogManager.getLogger(RdbmsUtils.class);
+  private static final String MAX_EPOCH_NUMBER_SQL_1 =
+      "select max(o.epoch_number) From (select keyvaluemapping_id, epoch_number from observation where experiment_run_id ='";
+  private static final String MAX_EPOCH_NUMBER_SQL_2 =
+      "' and entity_name = 'ExperimentRunEntity') o, (select id from keyvalue k where kv_key ='";
+  private static final String MAX_EPOCH_NUMBER_SQL_3 =
+      "' and  entity_name IS NULL) k where o.keyvaluemapping_id = k.id ";
```

Comment:
I think https://github.com/VertaAI/modeldb/pull/825/files#diff-0de1b97140d6153d08ac22f8ba1d58d0R101 does not work because the inner query and the outer query does not support the same alias. please verify it.

```suggestion
      "' and  entity_name IS NULL) kv where o.keyvaluemapping_id = kv.id ";
```

Reply:
it works, but valid point i removed the  inner